### ISSUE TITLE
Feature request: Set routing allocation to null when enabling allocation from cerebro

### ISSUE THREAD

**Issue by ceeeekay** [lmenezes/cerebro#566]
Hi there,

Recently Kibana has been complaining about routing allocation settings preventing index upgrades, even when they're in a state which was OK for previous Kibana versions.

`The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue`

My usual upgrade procedure on clusters without automation is to use the Cerebro padlock to set routing allocation to `new primaries`, upgrade & restart the node, and then unlock the padlock again when the node is back in the cluster.

As doing this sets routing allocation to `all`, Kibana gets upset and refuses to upgrade its index, because it's now expecting routing allocation settings to be absent. The fix for this is to manually set shard allocation to `null` to allow it to continue.

Tl;dr could we please have Cerebro set routing allocation to `null` instead of `all` when unlocking via the padlock icon?

**Comment by ceeeekay** [comment]
Kibana 8.5.x now seems to be accepting `all` when it migrates after a restart.
